high_card.rb

```diff
@@ -1,5 +1,89 @@
+require 'pry'
+
 # Use these two arrays to generate a deck of cards.
 ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
+# Create deck
+deck = []
+ranks.each { |rank| suits.each{ |suit| deck << ["#{rank}", "#{suit}"] } }
```

Comment:
if you make each `card` a hash it might make it easer when comparing high cards later on, especially if you add another property like `value` references each rank's index.